### DISCUSSION TITLE
Should we release 0.71.15?

### DISCUSSION THREAD

**Discussion by fortmarek** [reactwg/react-native-releases#90] [Patches]
0.71.15 has been released. See discussion for [0.71.16](https://github.com/reactwg/react-native-releases/discussions/99)

----

Current Release: [0.71.14](https://github.com/facebook/react-native/releases/tag/v0.71.14)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### Local commits to backport to main

1.

#### Pick requests from the previous discussion

- https://github.com/reactwg/react-native-releases/discussions/87#discussioncomment-7018982
- https://github.com/reactwg/react-native-releases/discussions/87#discussioncomment-7245123
- https://github.com/reactwg/react-native-releases/discussions/87#discussioncomment-7049821

**Comment by majkpal** [comment]
Please cherry-pick this fix: 
* [facebook/react-native@265af22](https://github.com/facebook/react-native/commit/265af222aa8acb3b514e22a7db191d66755f553b)


At Chime X Software Mansion we need this fix to be cherry-picked as currently there is no option to upgrade to the next version ( this fix has not been released anyway) and we are blocked with our next project there.

**Reply by hurali97** [reply]
Picked ✅

**Comment by cipolleschi** [comment]
Merge https://github.com/facebook/react-native/pull/41277 or you can't release as CI will stay red.

**Reply by hurali97** [reply]
Picked ✅

**Comment by liamjones** [comment]
It was suggested that the webp vulnerability patched in Fresco should probably be backported to supported versions of RN - just adding a note here for reference: https://github.com/react-native-community/discussions-and-proposals/issues/725#issuecomment-1780222618

**Reply by hurali97** [reply]
This is not valid anymore, context here: https://github.com/react-native-community/discussions-and-proposals/issues/725#issuecomment-1853987373

**Comment by linpengfei** [comment]
please cherry-pick this fix:
https://github.com/facebook/react-native/commit/94972039571e1f3b387e0f63227a6ad13740eaf3

**Reply by hurali97** [reply]
Picked ✅

**Comment by hurali97** [comment]
Need to merge this PR for CLI bump: https://github.com/facebook/react-native/pull/41478

**Reply by hurali97** [reply]
Picked ✅

**Comment by cipolleschi** [comment]
CircleCI got broken yesterday due to Ruby and bundler. This is the fix:
https://github.com/facebook/react-native/commit/479b4bb1ba504706613f71530a4ddcf86ff1bcd9

**Comment by hurali97** [comment]
This https://github.com/reactwg/react-native-releases/discussions/87#discussioncomment-7049821 will be picked in the next patch, since we are near holidays and releasing new hermes version isn't trivial at the moment.

**Comment by lunaleaps** [comment]
Merge https://github.com/facebook/react-native/pull/42106 to pin monorepo deps as we're doing this with 72, 73, and main. 

However, 71 doesn't fully have the monorepo deps fleshed out, also a lot of the packages have been re-named since so i'm not sure it's _that_ necessary to merge this. The only lib that I pinned was `react-native-codegen` which has been renamed and we aren't updating anyway. However, currently a `*` dep on `react-native-codegen` will pull in 0.70.7 when I'm assuming we want 0.71.6 (which is what I pinned in the PR)

**Reply by hurali97** [reply]
Picked ✅

**Comment by joe-sam** [comment]
Corrects LogBox to display the actual stack trace libraries instead of internal code on the debug thread for all custom errors during unhandled promises.
https://github.com/facebook/react-native/pull/42079

**Reply by lunaleaps** [reply]
@joe-sam Can you make a PR for this against `0.71-stable`? There are some merge conflicts

**Reply by joe-sam** [reply]
@lunaleaps I think you will have to first cherry pick this commit facebook/react-native#41377 onto 0.71-stable
  which will probably solve all the same merge conflicts before picking this. 
  I think this previous sequence fix has not yet been backported to 0.71.

**Reply by lunaleaps** [reply]
@joe-sam Do you mind making that PR with all the picks required -- I think the issue is also file restructuring that's happened since 0.71. @hurali97 will be releasing this next week Monday

**Reply by joe-sam** [reply]
Have created a combined pick PR as suggested. [#42613](https://github.com/facebook/react-native/pull/42163).

**Reply by hurali97** [reply]
Thanks for the effort and we will pick this in the next patch as we have to release a patch ASAP based on the issue: https://github.com/facebook/react-native/issues/42110

**Reply by scaraux** [reply]
Anyone knows when 0.71 next release is? I have the boost dep issue and can't bump to 0.72 for node engine version reasons...

**Reply by kelset** [reply]
@scaraux it's out: https://github.com/facebook/react-native/releases/tag/v0.71.15
